src/navigators/index.js

```diff
@@ -28,11 +31,33 @@ const AuthNavigator = createStackNavigator(
 const MainNavigator = createStackNavigator({
   MainScreen: {
     screen: MainScreen,
-    navigationOptions: {
+    navigationOptions: ({ navigation }) => ({
       title: strings.TITLE.main,
-      headerLeft: <Image source={profileIcon} style={styles.leftIcon} />,
+      headerLeft: (
+        <TouchableOpacity onPress={() => navigation.push(PROFILE_SCREEN)}>
+          <Image source={profileIcon} style={styles.leftIcon} />
+        </TouchableOpacity>
+      ),
       headerRight: <Image source={chatBubble} style={styles.rightIcon} />,
-    },
+      headerStyle: {
+        borderBottomWidth: 0,
+      },
+    }),
+  },
+  ProfileScreen: {
+    screen: ProfileScreen,
+    navigationOptions: ({ navigation }) => ({
+      headerLeft: null,
+      headerRight: (
+        <TouchableOpacity onPress={() => navigation.goBack()}>
```

Comment:
```suggestion
        <TouchableOpacity onPress={navigation.goBack}>
```
Only if this isn't passing parameters that cause unwanted side effects...

Reply:
yes, is that case so I leave it like this. But good to know for other cases.